google/data_source_stackdriver_ip_ranges_test.go

```diff
@@ -0,0 +1,30 @@
+package google
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceStackdriverIpRanges_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccStackdriverIpRangesConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestMatchResourceAttr("data.google_stackdriver_ip_ranges.some",
+						"cidr_ranges.#", regexp.MustCompile("^[1-9]+[0-9]*$")),
+					resource.TestMatchResourceAttr("data.google_stackdriver_ip_ranges.some",
+						"cidr_ranges.0", regexp.MustCompile("^[0-9./]+$")),
+				),
+			},
+		},
+	})
+}
+
+const testAccStackdriverIpRangesConfig = `
```

Comment:
Does this test need to be seeded with data in any way? I don't have anything stackdriver related in my project and it didn't pass:
```
$ make testacc TEST=./google TESTARGS='-run=TestAccDataSourceStackdriverIpRanges'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./google -v -run=TestAccDataSourceStackdriverIpRanges -timeout 120m
=== RUN   TestAccDataSourceStackdriverIpRanges_basic
--- FAIL: TestAccDataSourceStackdriverIpRanges_basic (0.86s)
	testing.go:435: Step 0 error: Check failed: Check 1/2 error: data.google_stackdriver_ip_ranges.some: Attribute 'cidr_ranges.#' didn't match "^[1-9]+[0-9]*$", got ""
FAIL
exit status 1
FAIL	github.com/terraform-providers/terraform-provider-google/google	1.030s
make: *** [testacc] Error 1
```

Reply:
Environment config:
```
$ gcloud auth login
$ gcloud auth application-default login
$ export GOOGLE_USE_DEFAULT_CREDENTIALS=1
$ export GOOGLE_PROJECT=terraform-tests-xxx
$ export GOOGLE_REGION=us-central1
$ export GOOGLE_XPN_HOST_PROJECT=terraform-tests-xxx
```

This seems to work:
```
$ make testacc TEST=./google TESTARGS='-run=TestAccDataSourceStackdriverUptimeCheckIps'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./google -v -run=TestAccDataSourceStackdriverUptimeCheckIps -timeout 120m
=== RUN   TestAccDataSourceStackdriverUptimeCheckIps_basic
--- PASS: TestAccDataSourceStackdriverUptimeCheckIps_basic (0.82s)
PASS
ok      github.com/terraform-providers/terraform-provider-google/google 0.831s
```

Reply:
This was a new GCP project and I didn't activate any additional services other than GCE.

I do have an existing Stackdriver account under my login, but this project was not linked to it yet.